ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -157,6 +157,9 @@ export default class SendAssetRow extends Component {
       if (nft) {
         return this.renderNft(nft);
       }
+      if (details) {
+        return this.renderNft(details);
+      }
```

Comment:
Thinking we could combine these two conditions

```suggestion
      if (nft || details) {
         return this.renderNft(nft ?? details);
      }
```

Reply:
__unresolving this__

now that they are combined, there is still a redundant 
```
if (nft) {
        return this.renderNft(nft);
}
```
check above

Reply:
This change is already added to code.